src/styles/index.scss

```diff
@@ -1,3 +1,22 @@
-body {
+@import 'fonts';
+@import 'variables';
+@import 'header';
+@import 'catalog';
+@import 'product-card';
+@import 'stars';
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: $fontRoboto;
+  font-size: 12px;
+  box-sizing: border-box;
+}
+
+.container {
+  width: fit-content;
```

Comment:
Using `fit-content` for the width might not be the best choice for a container. Consider using a more responsive approach, such as percentages or max-width, to ensure better adaptability across different screen sizes.